lib/v2/domain-shared/event_bus/events.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:meta/meta.dart';
+import 'package:seeds/v2/datasource/remote/model/transaction_model.dart';
+
+/// --- EVENT BUS EVENTS
+///
+@immutable
+abstract class BusEvent<T> {
+  const BusEvent();
+  @override
+  String toString() => 'EventBus { $T }';
+}
```

Comment:
No sure if there is a better way to avoid override the toString in each class. 
I mean I want to see the bus events in the console logs.

Reply:
It works for the class name but maybe not the best option if we want to see each field of that event.